cmd/internal/kube/kube.go

```diff
@@ -0,0 +1,15 @@
+package kube
+
+import (
+	"github.com/akitasoftware/akita-cli/cmd/internal/cmderr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "kube",
+	Short: "Gateway to Kubernetes related utilities",
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return cmderr.AkitaErr{Err: errors.New("no subcommand specified")}
+	},
```

Comment:
Maybe specify 'k8s' as an alias, or 'kubernetes'?

Reply:
I've added aliases for both in [935b4bb](https://github.com/akitasoftware/akita-cli/pull/202/commits/935b4bbcb6c6bb93fa562329ad22bce93911b09d)